lib/RedisRemoteSaiInterface.cpp

```diff
@@ -485,6 +493,114 @@ sai_status_t RedisRemoteSaiInterface::setRedisExtensionAttribute(
     return SAI_STATUS_FAILURE;
 }
 
+bool RedisRemoteSaiInterface::isSaiS8ListValidString(const sai_s8_list_t &s8list, const char* hint)
```

Comment:
Missing _ in _ in arrguments, also one argument per line

Reply:
fixed